Library/Formula/svmlight.rb

```diff
@@ -0,0 +1,16 @@
+class Svmlight < Formula
+  homepage "http://svmlight.joachims.org/"
+  url "http://download.joachims.org/svm_light/current/svm_light.tar.gz"
```

Comment:
Basically just a versioned URL so e.g. when a new version is uploaded it won't break the `sha1` of the previous download link. There's also no way of identifying that this is version 6.02 from the URL alone.